repository-data/webfiles/src/main/resources/site/freemarker/common/searchresults.ftl

```diff
@@ -11,17 +10,7 @@
             <h1 data-uipath="ps.search-results.count">${pageable.total} result<#if pageable.total gt 1>s</#if> found</h1>
             <h4 class="push-double--bottom">Displaying  ${pageable.startOffset +1 } to ${pageable.endOffset} of ${pageable.total} results for '${query}'</h4>
 
-            <#list pageable.items as publication>
-                <div class="push-double--bottom" data-uipath="ps.search-results.result">
-                    <p class="flush">
-                        <a href="<@hst.link hippobean=publication.selfLinkBean/>" title="${publication.title}" data-uipath="ps.search-results.result.title">
-                            ${publication.title}
-                        </a>
-                    </p>
-                    <p class="flush zeta" data-uipath="ps.search-results.result.date"><@formatRestrictableDate value=publication.nominalPublicationDate/></p>
-                    <p class="flush" data-uipath="ps.search-results.result.summary"><@truncate text=publication.summary.firstParagraph size="300"/></p>
-                </div>
-            </#list>
+            <#include "./macro/results.ftl">
```

Comment:
Hmm, I'd suggest that:
* the `<#list...` block in the `results.ftl` be wrapped in a macro
* this `<#include...` moved to the top of the file
* make this line call the macro rather than include.

I see 'includes' more as a declaration of some re-usable functionality rather than the application of it. To my mind, this is similar to Java imports or C headers' inclusions - at the beginning of the file we declare what we will be used and later in the body we actually use it by calling individual macros defined in the included files.

Also, having all reusable content wrapped in macros makes it easier to pick-and-choose individual macros from the included file; conversly, having a bit of inlined content (i.e. not wrapped in a macro) in the included file forces us to use it as soon as we include the file, which may or may not be applicable in all situations.

I emphasise that this is just my subjective view so feel free to ignore it - but I think this keeps the files a bit more 'predictable' and easier to read and reusable.